### DISCUSSION TITLE
Elsa 2.2.1 webhook definitions error

### DISCUSSION THREAD

**Discussion by parsec99** [elsa-workflows/elsa-core#1384] [Q&A]
I migrated my project from Elsa 2.0 to Elsa 2.2.1 by recreating the database and it works well except the tab 'WebHook Definitions' in the dashboard.  I don't use webhooks for now but maybe in the future. :-)

I got an error 500 when I clicked on the button 'webhook definitions' :

![webhook-definitions-error](https://user-images.githubusercontent.com/5891353/129174475-bfe3aa74-5143-4499-8a14-072eb2baf83d.png)

With Swagger, I executed the api /v1/webhook-definitions and I got a more detailed error :

![webhook-definitions-swagger-error](https://user-images.githubusercontent.com/5891353/129174708-db1dd888-6f52-4148-8641-3f002796c4ef.png)

Is this a known bug or is it only me?

Thanks for your help.

**Comment by sfmskywalker** [comment]
Hi @damienheckler ,

This feature is currently in flux. Although it's visible from the dashboard, you need to actually install the back-end bits explicitly for the front-end to work.

We're working on making the front-end more modular so that you need to opt-in there as well. The result being that you would no longer even see the webhook definition menu items.

/cc @Axeleron007

**Reply by parsec99** [reply]
Ok, I understand. 😊
Thank you @sfmskywalker.
